packages/sdk/.nycrc

```diff
@@ -1,19 +0,0 @@
-{
-    "include": [
-      "dist-esm/src/**/*.js"
```

Comment:
what config will be used after we remove this file?

Reply:
I think it will use the default configuration of nyc which is similar to other packages.

Reply:
Seems by default nyc only includes files that were required: https://www.npmjs.com/package/nyc#selecting-files-for-coverage.
So if we add a new file but don't add test case for it, the code coverage won't reduce. This might not be the expected result. Not sure whether other default configuration meets our requirement.

Reply:
Thank you! I agree that it is not expected, adding @LongOddCode for awareness.

Reply:
Added the nycrc file back. @blackchoey